### DISCUSSION TITLE
Metalama 2023.3.5-rc released

### DISCUSSION THREAD

**Discussion by addabis** [metalama/Metalama#218] [Changelog]
We've just released Metalama 2023.3.5-rc which includes all fixes and enhancements from [Metalama 2023.1.12](https://github.com/orgs/postsharp/discussions/216) and [Metalama 2023.2.5](https://github.com/orgs/postsharp/discussions/217) in addition to following changes.

### New
* You can now use the C# Dev Kit extension for VS Code and get Metalama design-time experience.

### Improved
* Performance of syntax highlighting in large projects was improved.

### Fixed
* Fixed: CompileTimeCompilationBuilder throws IOException when moving the PE file. _This occurred on CI agents after the automatic cleanup could not remove the file, corrupting the cache._
* Fixed: Can't get enhancements applied to a generic base of the current target.
* Fixed: Aspect class derived from OverrideFieldOrPropertyMulticastAspect fails to compile.
* Fixed: Introducing an attribute when no attributes currently exist places the attribute above comments.
* Fixed: AspectTests: AssemblyMetadataAttributes are missing when running from VS.
* Fixed: Metalama.Compiler does not start the compiler process on Linux/Mac. _This was introduced when merging the latest Roslyn revision and affected the previous two preview releases._
* Fixed: CompileTimeErrorsInfoBarService throws ArgumentNullException when there is an error without a file path.
* Fixed: Test framework: XmlDocumentationReader throws NullReferenceException in case of missing cref attribute.
* Fixed: Template compiler: cast exception when transforming "for" statement.
* Fixed: Test framework: test output does not appear in Rider

**Comment by gfraiteur** [comment]
The VSX is published on [OneDrive](https://1drv.ms/u/s!AjIdLvQsWyhnhop_nzI_-Hdb9esOWA?e=ZoGDCq). This is a highly recommended update.
